github_tutorial.md

```diff
@@ -94,67 +97,83 @@ De beheerder van de standaard zal dus de volgende handelingen doen voor een vers
 	* Klik op "Propose new file"
 
 2. Ik wil een (tekst)document wijzigen
-	* Op en het document vanuit tab "Code" door te klikken op de bestandsnaam
+	* Ga naar de tab "Code"
+	* Kies daar de branch waarin je de wijzigingen wil aanbrengen
+	* Open het document dat je wil aanpassen door te klikken op de bestandsnaam
 	* Klik op het pen-icoon
-	* Voor de toevoegingen of wijzigingen door in de inhoud van het document
-	* Vul bij "Propose new file" een korte titel voor de toevoegingen of wijzigingen en een uitgebreide toelichting hiervan
-	* Klik op "Propose file change"
-
-3. Ik wil een opmerking maken over een bijdrage van iemand anders
+	* Voer de toevoegingen of wijzigingen door in de inhoud van het document
+	* Commit de wijzigingen in het document door op "Commit changes" te klikken waarbij je de radio button "Commit directly to the nnnnn branch." selecteert. "nnnnn" staat voor de branch waarin je deze wijziging wilde aanbrengen.
+
+3. Ik wil een "Pull request" indienen
+	* Open de tab "Pull requests"
+	* Klik op "New pull request"
+	* Selecteer bij "base fork" de repository en bij "base" de branch waar het pull request op moet worden geplaatst
+	* Selecteer bij "head fork" de repository en bij "compare" de branch waarin de wijziging is aangebracht
+	* Klik op "Create pull request" en ken er een naam aan toe die voldoet aan de volgende conventie:
```

Comment:
Zo had ik het niet bedoeld. De conventie is dat de naam van de pull request beschrijft wat de bedoeling ervan is.
Maar er is de mogelijkheid om:
1. Wanneer de pull request een issue helemaal oplost dit issue te sluiten via Fixes #.. (dus niet Fixit maar Fixes)
2. Wanneer een pull request betrekking heeft op (een deel van) een issue, hiernaar te refereren in de naam met Ref. #..

Bijvoorbeeld: "Ref. #6 Toevoegen aan tutorial van mogelijkheid te verwijzen naar een issue in een pull request"
Bijvoorbeeld: "Fixes #6 Verduidelijken en corrigeren in tutorial van werkwijze indienen pull request"

Reply:
Ok, ik meende dat het accepteren van de pull request al meteen zou leiden tot het sluiten van het issue. Als ik het goed begrijp is het echter alleen het bieden van een link naar het issue om dit daar dan snel te kunnen sluiten. Of begrijp ik het dan nog niet goed?

Reply:
Ga ik aanpassen.